circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:
+        parallel: true
+    - pip install -r docs-requirements.txt:
+        parallel: true
+    - rm -rf docs/API && make -C docs clean:
+        parallel: true
     - python setup.py docs:
         parallel: true
```

Comment:
And I'm not familiar with CircleCI :D So if you can tell me how to restrict steps under `test:` to certain nodes, I would be very happy...